requirements/requirements_lightning.txt

```diff
@@ -1,7 +1,7 @@
 cloudpickle
 fiddle
 hydra-core>1.3,<=1.3.2
-lightning>2.2.1
+lightning>2.2.1,<=2.4.0
```

Comment:
Are you addressing the progress printer issue with 2.5? You can fix that directly by calling `on_train_epoch_start` from inside of a new `on_fit_start` in the progressprinter class.

Reply:
Eg you could add:

```python
    @override
    def on_fit_start(self, trainer: Trainer, pl_module: LightningModule) -> None:
        self.on_train_epoch_start(trainer, pl_module)
```

Right before https://github.com/NVIDIA/NeMo/blob/main/nemo/lightning/pytorch/callbacks/progress_printer.py#L114-L120

Reply:
that got 2.5 working for me with job resumption without the `AttributeError: 'ProgressPrinter' object has no attribute 'total'` error.

Reply:
Virtually all tests are failing. Thats why . Needs workaround before 2.5 upgrade